lrpar/src/lib/parser.rs

```diff
@@ -581,9 +581,9 @@ pub(crate) mod test {
         for l in lexs.split("\n").map(|x| x.trim()).filter(|x| !x.is_empty()) {
             println!("{}", l);
             assert!(l.rfind("'") == Some(l.len() - 1));
-            let name_beg = l[..l.len() - 1].rfind("'").unwrap();
-            let name = &l[name_beg + 1 .. l.len() - 1];
-            let re = &l[..name_beg - 1].trim();
+            let name_st = l[..l.len() - 1].rfind("'").unwrap();
```

Comment:
I know you love short variable names, but would it really hurt so much to rename this to `name_start` so that it's absolutely clear what this variable contains and readers don't have to guess what `st` means?

Reply:
In retrospect, "start" is a bit misleading, because it's the position of the input after the opening `'`. So a truthful variable name would be something like `first_character_of_name_after_quote`. It might be clearer to just give it a meaningless name like `i`?

Reply:
Hah, ye maybe a meaningless name is better in this instance. ;-)